src/js/stores/DCOSStore.js

```diff
@@ -440,7 +440,7 @@ class DCOSStore extends EventEmitter {
       .getServiceList()
       .reduceItems(function(memo, framework) {
         if (framework instanceof Item) {
-          memo[framework.get("name")] = framework.get();
+          memo[`/${framework.get("name")}`] = framework.get();
```

Comment:
Is this just a cleanup, or was the `/` missing and required somewhere else?

Reply:
Have a look at the change down below. `/` was required to make them equal